_episodes/02-markdown.md

```diff
@@ -119,123 +119,82 @@ Let's do some exercises to learn more about structuring and emphasis.
 
+Displaying items in lists, sometimes increases readability.
+You can create lists by preceding lines with `-` or a bunch of other `-`-like characters.
 
-> ## Lists, Enumerations
+> ## List today's learning goals
 >
-> Try to reproduce the source code of this challenge from the rendered view below this line:
+> Give it a try and write a short list of topics you want to learn today.
 >
-> ---
->
-> - A list of items
-> - can be
->   - deeply
->     - nested
-> - returning to the top-level like this.
+> > ## Solution
+> >
+> >     - Write Markdown
+> >     - Host my first website on GitHub
+> >     - Learn about Jekyll
+> >     - Think about how my work group could benefit from what I have learned.
+> >
+> {: .solution }
+{: .challenge }
+
+If you would like to nest lists, i.e. specify a sub list as an item of another list, use indentation.
+Numbered enumerations are helpful for displaying sequences and the numbers may help to address individual points.
+
+> ## Nested Lists and Enumerations
 >
-> New text blocks require **blank lines** and
+> Try to reproduce the source code of the following view
 >
-> 23. enumerations
-> 23. starting at any number
-> 23. continue to increase
-> 26. in steps of one.
->     1. Sub enumerations (in some editors have to start at **one**)
->     2. and can include
->        - sub lists
->     4. keeping the enumeration count.
+> ![Preview tab nested lists and enumerations](../fig/exercise_nested_lists_enumerations.png)
 >
 > > ## Solution
-> > ~~~
-> > - A list of items
-> > - can be
-> >   - deeply
-> >     - nested
-> > - returning to the top-level like this.
 > >
-> > New text blocks require **blank lines** and
+> >     - A list of items
+> >     - can be
+> >       - deeply
+> >         - nested
+> >     - returning to the top-level like this.
 > >
-> > 23. enumerations
-> > 23. starting at any number
-> > 23. continue to increase
-> > 26. in steps of one.
-> >     1. Sub enumerations (in some editors have to start at **one**)
-> >     2. and can include
-> >        - sub lists
-> >     4. **without** disturbing the enumeration count.
-> > ~~~
-> > {: .source }
+> >     2. Enumerations can start at any number and
+> >     3. continue to increase
+> >     26. in steps of one even of the numering does not. This updates the numbering if individual items are deleted.
 > {: .solution }
 {: .challenge }
 
+> ## Markdown Cheatsheet
+>
+> Markdown offers a varyity of formatting features.
+> Have a look at this [cheatsheet][github-flavored-markdown] to get an overview or look things up.
+>
+{: .callout}
 
-
+You can use single backticks to call out code, e.g. `print('Hello world')` and
+use triple backticks to format larger code snippets, creating code blocks.
 
 > ## Add Code
-> Fill in the blanks below to create the rendered view below this line:
+> Fill in the blanks ( __ ) below to create the following view
 >
-> ---
+> ![Preview tab with blanks before code and code blocks](../fig/exercise_add_code.png)
```

Comment:
I'm confused about why the preview includes the blanks? Shouldn't this exercise include a block of Markdown with blanks, and a preview of how the completed Markdown should render, after the blanks have been filled in?

Reply:
Thanks, I thought this exercise could do without code-template, but your confusion proofed that I was wrong.
I adopted the format used in the other "fill the blanks" exercises in 
7bb3b53  and agree, that this is much better.